### DISCUSSION TITLE
Welcome to Wifi-Hacking Discussions!

### DISCUSSION THREAD

**Discussion by ankit0183** [ankit0183/Wifi-Hacking#49] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Exparte5159** [comment]
Hi I'm new to ethical hacking and need advice on what tools to setup in termux

**Reply by sabarees3578** [reply]
Use Kali Linux bro all the tools inbuilt

**Reply by jvlsean** [reply]
well Kali is too expert I usually use ubuntu

**Comment by Hasanq2** [comment]
Hack

**Comment by ghost** [comment]
although the readme says blackarch was tested it uses pacman no ? so the script would try to use apt instead of pacman am i dense?

**Comment by jvlsean** [comment]
I'll be going to a competition need help to understand this rq.

**Comment by Nagaraju3737** [comment]
Brother can anyone please tell me how to use this repo for wifi hacking.

**Reply by Nagaraju3737** [reply]
i am bit confusd...🫤
